### PR TITLE
Fix cases of method named with get

### PR DESCRIPTION
**Pull request by natebosch**
https://dart.dev/guides/language/effective-dart/design#avoid-starting-a-method-name-with-get

### CODE REVIEW

**Comment by natebosch** [comment]
I don't know where  to find the definition of the `getRole` method mentioned here: https://github.com/dart-lang/site-www/pull/2134/files#diff-7ad8781f760aea67193435d1aeafae8fR356

Is that in another repo? What do we need to do to coordinate fixing this one?

**Comment by natebosch** [comment]
Do we have any Dart readability experts reviewing our codelabs before they are submitted? If not, it might be a good idea to start doing that.

cc @munificent @vsmenon @legalcodes

File: src/tools/sdk/sdk_builds.dart/lib/src/dart_downloads.dart [hunk 1]
```diff
@@ -36,7 +36,7 @@ class DartDownloads {
     return new DartDownloads._(client);
   }
 
-  Future<Uri> getDownloadLink(
```
Comment (kwalrath) [inline src/tools/sdk/sdk_builds.dart/lib/src/dart_downloads.dart]: Hmmm... "download" sounds like a verb, but I can't think of a better name. (In fact, I thought all function names were supposed to start with verbs... but TIL [that's not true](http://localhost:4000/guides/language/effective-dart/design#prefer-a-noun-phrase-or-non-imperative-verb-phrase-for-a-function-or-method-if-returning-a-value-is-its-primary-purpose).)

**Comment by chalin** [comment]
> I don't know where to find the definition of the getRole method mentioned here: https://github.com/dart-lang/site-www/pull/2134/files#diff-7ad8781f760aea67193435d1aeafae8fR356

Currently, it is in this Gist: https://gist.github.com/legalcodes/f751b692502c4ee43d932f745860b056

FYI, PR #2114 is moving the code from the Gists into the repo examples folder.

**Comment by natebosch** [comment]
> FYI, PR #2114 is moving the code from the Gists into the repo examples folder.

I'd be happy to wait until that is merged and perform the renames again.

File: src/tools/sdk/sdk_builds.dart/lib/src/dart_downloads.dart (hunk 1 above)
Comment (natebosch) [reply src/tools/sdk/sdk_builds.dart/lib/src/dart_downloads.dart]: That's true, this name does make it sound like a verb. How about `createDownloadUrl`?

**Comment by legalcodes** [comment]
@natebosch Agreed we should have readability review before we submit codelab content.

File: src/tools/sdk/sdk_builds.dart/lib/src/dart_downloads.dart (hunk 1 above)
Comment (kwalrath) [reply src/tools/sdk/sdk_builds.dart/lib/src/dart_downloads.dart]: sgtm

**Comment by kwalrath** [comment]
@legalcodes can you coordinate with @natebosch on the fixes to the codelab code, or should he back out the codelab changes for now?

**Comment by legalcodes** [comment]
@kwalrath Yes, happy to coordinate with @natebosch on these fixes.

**Comment by natebosch** [comment]
I'll revert the codelab changes for now to unblock the rest of the changes.

**Pull request merged**